### PR TITLE
Otlp exporter doc update

### LINKED ISSUE #4026: How do I change the export frequency of the OTLPExporter?
# Question

Currently I am attempting to set the OTLPExporterOptions for a MetricsProvider that relate to export frequency. Using the following I am able to successfully set up a provider and export metrics at the default time period of 10 secs (10000 ms):

MeterProvider metricsProvider = Sdk.CreateMeterProviderBuilder()
                    .AddMeter(serviceName)
                    .AddConsoleExporter()
                    .AddOtlpExporter(opt =>
                    {
                        opt.ExportProcessorType = ExportProcessorType.Simple;
                        opt.Endpoint = new Uri(metricsURI);
                        opt.Protocol = OtlpExportProtocol.Grpc;
                    })
                    .Build();

**My question is, how is it possible to change the 10 second interval for this export to a collector?**

I have tried several approaches:

1. Setting the environment variables listed here https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md does not affect the export time.

2. Setting the timeout in code via `opt.TimeoutMilliseconds` also does not affect the export time.

3. Setting the timeout via `opt.BatchExportProcessorOptions.ScheduledDelayMilliseconds`. This appeared to be the wrong setting via this question https://github.com/open-telemetry/opentelemetry-dotnet/issues/2767 and also does not affect the export time.

.NET version: net6.0
Packages:
OpenTelemetry Version=1.3.1
OpenTelemetry.Exporter.Console Version=1.3.1
OpenTelemetry.Exporter.OpenTelemetryProtocol Version=1.3.1

### PR DESCRIPTION
**Pull request by vishweshbankwar**
Towards #4417
Fixes #4026 

Design discussion issue #

## Changes

Please provide a brief description of the changes here.

## Merge requirement checklist

* [ ] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (nullable enabled, static analysis, etc.)
* [ ] Unit tests added/updated
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes
* [ ] Changes in public API reviewed (if applicable)

### CODE REVIEW

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 1]
```diff
@@ -46,8 +46,15 @@ var meterProvider = Sdk.CreateMeterProviderBuilder()
     .Build();
 ```
 
-See the [`TestMetrics.cs`](../../examples/Console/TestMetrics.cs) for
-runnable example.
+By default, `AddOtlpExporter()` configures the metric exporter as
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:49]:
```suggestion
By default, `AddOtlpExporter()` pairs the OTLP MetricExporter with a
```

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 2]
```diff
@@ -46,8 +46,15 @@ var meterProvider = Sdk.CreateMeterProviderBuilder()
...
+[PeriodicExportingMetricReader](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#periodic-exporting-metricreader)
+with
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:51]:
```suggestion
with metric export interval of 60 secs and 
```

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 3]
```diff
@@ -122,10 +129,12 @@ The following environment variables can be used to override the default
...
-| Environment variable          | `PeriodicExportingMetricReaderOptions` property |
-| ------------------------------| ------------------------------------------------|
-| `OTEL_METRIC_EXPORT_INTERVAL` | `ExportIntervalMilliseconds`                    |
-| `OTEL_METRIC_EXPORT_TIMEOUT`  | `ExportTimeoutMilliseconds`                     |
+| Environment variable                                | `PeriodicExportingMetricReaderOptions` property |
+| ----------------------------------------------------| ------------------------------------------------|
+| `OTEL_METRIC_EXPORT_INTERVAL`                       | `ExportIntervalMilliseconds`                    |
+| `OTEL_METRIC_EXPORT_TIMEOUT`                        | `ExportTimeoutMilliseconds`                     |
+
+NOTE:  `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` is not supported yet.
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:137]: nit : add link to the tracking issue.

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 4]
```diff
@@ -46,8 +46,15 @@ var meterProvider = Sdk.CreateMeterProviderBuilder()
     .Build();
 ```
 
-See the [`TestMetrics.cs`](../../examples/Console/TestMetrics.cs) for
-runnable example.
+By default, `AddOtlpExporter()` pairs the OTLP MetricExporter with a
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:49]: Ideally we just document defaults in the "## Configuration" section for all 3 signals. But that can be a future re-factoring.

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md (hunk 4 above)
Comment (vishweshbankwar) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:49]: Agree. There are some changes coming to logs and traces. Once those are done, we can update the configuration section. I just added this for now as we get common ques around how Otlp metric exporter is configured?

**Review by utpilla — APPROVED** [review]

**Pull request merged**